src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -125,6 +125,14 @@ class DataCondition(DefaultFieldsModel):
         on_delete=models.CASCADE,
     )
 
+    def get_snapshot(self) -> dict[str, Any]:
```

Comment:
where is this used?

Reply:
https://github.com/getsentry/sentry/pull/91246/files#diff-920f779b1cdc3aa886dfcacf9ed9bf28340ab462500ee39260dfb21d96245fcaR308